etc/conda3_packages-linux-64.txt

```diff
@@ -1,66 +1,84 @@
 # This file may be used to create an environment using:
 # $ conda create --name <env> --file <this file>
 # platform: linux-64
-asn1crypto=0.22.0=py36_0
-astropy=2.0.1=np113py36_0
-bottleneck=1.2.1=np113py36_0
-certifi=2016.2.28=py36_0
-cffi=1.10.0=py36_0
-cryptography=1.8.1=py36_0
-cycler=0.10.0=py36_0
-cython=0.26=py36_0
-dbus=1.10.20=0
-expat=2.1.0=0
-fontconfig=2.12.1=3
-freetype=2.5.5=2
+asn1crypto=0.24.0=py36_0
+astropy=3.0.3=py36h14c3975_2
```

Comment:
Have you tested this configuration? I'm pretty sure that astropy 3.0.3 breaks everything (DM-14525)

Reply:
Not even remotely close to running `validate_drp` -- I guess we'll have to put a constraint in the bleed file (maybe bleed is a poor choice of name...).

Reply:
or have someone fix `validate_drp`...

Reply:
I think it was matplotlib that was the problem with DM-14525 and 2.2.2 (which we are using here) fixes the problem.